examples/java7/build-java7.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+which acbuild || (echo "Make sure acbuild is on your path"; exit 1)
+
+TMPDIR=$(mktemp -d)
+PACKAGESDIR=$(mktemp -d)
+
+rmDirectories() {
+    rm -rf ${TMPDIR}
+    rm -rf ${PACKAGESDIR}
+}
+trap rmDirectories EXIT
+
+pushd ${PACKAGESDIR}
+apt-get download \
+    ca-certificates-java \
+    java-common \
+    libasound2 \
+    libatk1.0-0 \
+    libatk-wrapper-java-jni \
+    libc6 \
+    libcairo2 \
+    libcups2 \
+    libfontconfig1 \
+    libfreetype6 \
+    libgcc1 \
+    libgdk-pixbuf2.0-0 \
+    libgif4 \
+    libglib2.0-0 \
+    libgtk2.0-0 \
+    libkrb5-3 \
+    liblcms2-2 \
+    libnss3 \
+    libpango-1.0-0 \
+    libpangocairo-1.0-0 \
+    libpangoft2-1.0-0 \
+    libpcsclite1 \
+    libpng12-0 \
+    libpulse0 \
+    libstdc++6  \
+    libx11-6 \
+    libxext6 \
+    libxi6 \
+    libxinerama1 \
+    libxrandr2 \
+    libxrender1 \
+    libxtst6 \
+    openjdk-7-jre \
+    openjdk-7-jre-headless \
+    tzdata-java \
+    zlib1g \
+    bash \
+    libncurses5 \
+    libtinfo5
+popd
+
+shopt -s nullglob
+for d in ${PACKAGESDIR}/*.deb; do
+  echo "Extracting $d"
+  dpkg-deb -x ${d} ${TMPDIR}
```

Comment:
I tried using debootstrap to make a base, and without installing anything on top of that the ACI is at 134 MB.

I tried using the instructions in [this SO post](https://stackoverflow.com/questions/3954584/running-apt-get-for-another-partition-directory) to just `apt-get install` nginx into a different root dir. It pulled in so many packages (packages like apt, adduser, mount, e2fslibs, ...) that it would use up 163 MB of disk space, and ended up erroring on the install process when `/usr/bin/apt-listchanges` wasn't found.

Just extracting deb files into a directory skips potentially important install steps, and doing it correctly signs you up for containers greater in size than 134 MB. I really think the only decent solution to integrating a package manager with a container build system is nix2aci, where you have a package manager that doesn't need to include everything and the kitchen sink in case there are undeclared dependencies (nix is guaranteed to actually know the dependencies of its packages, instead of "oh the guy had this binary available when he built the package so you need to too").

So @jonboulle, want me to leave this in its current state, make the containers massive, or drop this and just point to nix2aci?